src/convertToObject.js

```diff
@@ -3,16 +3,42 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+
+  sourceString
+    .trim()
```

Comment:
```suggestion

  const splittedString = sourceString.split(';');
  
  return splittedString.reduce((acc, item) => {
    // convert styles logic
  }, {})
```
suggest simplify solution. 
no need to replace a lot of time each cases.
just split by `;` and make all logic in reduce method.
you will need only split and trim methods in reduce callback logic.